docs/setup.rst

```diff
@@ -16,6 +16,20 @@ You can get the library directly from PyPI:
 
 .. note:: Raster Vision requires Python 3 or later.
 
+macOS Installation
```

Comment:
I did not have to do this for my MacOS installation, for whatever reason - it could be because the necessary libraries were installed with other dependencies previously. Considering that, should we frame this more as a "if you have the following issues, you might need to do this" rather than as the definitive Mac install instructions? That way we don't encourage folk to install an unreleased version of pyproj if they don't have to.

I think another thing besides Mac that's worth maybe mentioning is that it's a python 3.7 issue (which was mentioned in the pyproj  issues you linked to in #487)

Reply:
Ah, okay -- that makes sense. If we're okay keeping the header, I can change it to something like...

##### Troubleshooting macOS Installation

If you encounter problems running `pip install rastervision` on macOS, you may have to manually install Cython and pyproj. To circumvent an issue installing pyproj with Python 3.7, you may also have to install it using `git+https`:

... instructions

##### Tensorflow header

....

Reply:
I think that makes sense. Thanks!

Reply:
Cool. Updated and squashed!